documentation/src/asciidoc/setting-up.adoc

```diff
@@ -152,3 +152,82 @@ class CalculatorSpec: Spek({
 IMPORTANT: As mentioned in the <<ide-support>> section, the IDEA plugin won't work if you're using the JUnit 4 runner.
 
 
+
+[[setting-up-android]]
+== Setting up Android
```

Comment:
As a matter of fact it did not help us using AS 3.0 RC 1.

Also, specify the AS versions supported by this.

Reply:
Is it safe to assume almost everybody is now using AS 3.0+?

Reply:
Not even close unfortunately (or otherwise). Especially noting that it is still a preview.

Reply:
Okay, the only thing that is specific to AS 3.0+ is point `7` - I'll include it there.